index.d.ts

```diff
@@ -0,0 +1,207 @@
+import * as cssColors from 'color-name';
+
+declare namespace ansiStyles {
+	interface ColorConvert {
+		/**
+		The RGB color space.
+
+		@param red - (0-255).
+		@param green - (0-255).
+		@param blue - (0-255).
+		*/
+		rgb(red: number, green: number, blue: number): string;
+
+		/**
+		The HSL color space.
+
+		@param hue - (0-360).
+		@param saturation - (0-100).
+		@param lightness - (0-100).
+		*/
+		hsl(hue: number, saturation: number, lightness: number): string;
+
+		/**
+		The HSV color space.
+
+		@param hue - (0-360).
+		@param saturation - (0-100).
+		@param value - (0-100).
+		*/
+		hsv(hue: number, saturation: number, value: number): string;
+
+		/**
+		The HSV color space.
+
+		@param hue - (0-360).
+		@param whiteness - (0-100).
+		@param blackness - (0-100).
+		*/
+		hwb(hue: number, whiteness: number, blackness: number): string;
+
+		/**
+		The CMYK color space.
+
+		@param cyan - (0-100).
+		@param magenta - (0-100).
+		@param yellow - (0-100).
+		@param black - (0-100).
+		*/
+		cmyk(cyan: number, magenta: number, yellow: number, black: number): string;
+
+		/**
+		The XYZ color space.
+
+		@param x - (0-100).
+		@param y - (0-100).
+		@param z - (0-100).
+		*/
+		xyz(x: number, y: number, z: number): string;
+
+		/**
+		The LAB color space.
+
+		@param x - (0-100).
+		@param y - (0-100).
+		@param z - (0-100).
+		*/
+		lab(lightness: number, a: number, b: number): string;
+
+		/**
+		The LCH/HCL color space.
+
+		@param lightness - (0-100).
+		@param chroma - (0-100).
+		@param hue - (0-360).
+		*/
+		lch(lightness: number, chroma: number, hue: number): string;
+
+		/**
+		The RGB HEX color space.
+
+		@param hex - A hexadecimal string containing RGB data.
+		*/
+		hex(hex: string): string;
+
+		/**
+		@param keyword - A CSS color name.
+		*/
+		keyword(keyword: keyof typeof cssColors): string;
+
+		ansi(ansi: number): string;
+		ansi256(ansi: number): string;
```

Comment:
These are missing comments. See https://github.com/chalk/chalk/commit/fb8e85ab875eb95b55448fd615daacf8c325cf44 and https://github.com/chalk/chalk/pull/369